features/invitelist/impl/src/main/kotlin/io/element/android/features/invitelist/impl/components/InviteSummaryRow.kt

```diff
@@ -145,17 +145,17 @@ internal fun DefaultInviteSummaryRow(
                 OutlinedButton(
                     content = { Text(stringResource(StringR.string.action_decline), style = ElementTextStyles.Button) },
                     onClick = onDeclineClicked,
-                    modifier = Modifier.weight(1f),
-                    contentPadding = PaddingValues(horizontal = 24.dp, vertical = 7.dp),
+                    modifier = Modifier.weight(1f).heightIn(max = 36.dp),
```

Comment:
I believe this min `40.dp` is because Material Design components are supposed to have a minimum touch target size of 40px, but if the design team gave it the OK it's not that much of a difference.

Reply:
Yeah. Fortunately compose expands the touch target automagically if the component is smaller than `48dp`.

This lets you tap just above or below the button to trigger it:

![image](https://github.com/vector-im/element-x-android/assets/189133/f06052bd-f3e9-4133-ac0a-a159fb7920d1)

so it basically acts like a 48dp high button  while still conforming to the designs